flink-ml-parent/flink-ml-lib/src/main/java/org/apache/flink/ml/common/statistics/basicstatistic/BaseSummarizer.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.flink.ml.common.statistics.basicstatistic;
+
+import org.apache.flink.ml.common.linalg.DenseMatrix;
+
+import java.io.Serializable;
+
+/**
+ * Summarizer is the base class to calculate summary and store , and Summary is the result of Summarizer.
+ * It will compute sum, squareSum = sum(x_i*x_i), min, max, normL1.
```

Comment:
better to explain what the base class do. because this base class does not provide any of these functionalities.
```
Base Summarizer class holds basic stats (element count, outerproduct, etc) 
```

Reply:
Thanks, I have refined the JavaDoc.

Reply:
Thanks, we have refined the JavaDoc.